components/mpas-ocean/src/mode_forward/mpas_ocn_time_integration_rk4.F

```diff
@@ -172,6 +174,9 @@ subroutine ocn_time_integrator_rk4(domain, dt)!{{{
       type (field2DReal), pointer :: normalVelocityField, layerThicknessField
       type (field3DReal), pointer :: tracersGroupField
 
+      ! Time of day
+      real (kind=RKIND), pointer :: daysSinceStartOfSim
```

Comment:
note that this variable is also available from the ocn_diagnostics_variables module included above.  It's not clear to me though if it will be incremented, but assume it will.  It might be good to test without the access of the diagnostic pool every timestep